### DISCUSSION TITLE
Unsupported key in configuration file: [odoh_servers]

### DISCUSSION THREAD

**Discussion by Devices** [DNSCrypt/dnscrypt-proxy#1920] [Q&A] [answered]
Whenever I try to start dnscrypt-proxy I get this error, despite not having configured any sort of key. Other than odoh_servers = false, there is not a single mention of 'odoh' and definitely no [odoh_servers] section in the configuration.

My current configuration: https://pastebin.com/fsTQ0s5r

I am at a complete loss, to be honest. Anyone any idea?

Running 2.0.45+ds1-1 on debian bullseye

**Comment by welwood08** [comment, marked as answer]
ODoH support was first added in 2.0.46-beta1 which became 2.1.0, since you are using an older version you should start with the `example-dnscrypt-proxy.toml` included in that installation rather than anything from a more recent version.

Regarding the `Unsupported key` error, it's not complaining about any `[odoh_servers]` section. The line `odoh_servers = false` refers to the key `odoh_servers` whose value is `false`, and that is unsupported in the version you're using.

**Reply by Devices** [reply]
That makes me feel very silly. Sorry for wasting your time.
